### DISCUSSION TITLE
Embed captions into files during export

### DISCUSSION THREAD

**Discussion by vishnukvmd** [ente-io/ente#601] [Enhancements]
Currently captions / descriptions are written into a separate JSON sidecar file during export. This might be necessary for certain file formats where this data cannot be embedded into the original file. But for file formats that do support embedding this information, we should do so during export and save users from the trouble of patching this information.

**Comment by alnetloc** [comment]
👍🏼

**Comment by olfway** [comment]
But probably that could be made optional, so it's up to user if he want unmodified files or not
